helm-charts/seldon-core-controller/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Seldon Core CRD and controller helm chart for Kubernetes
+keywords:
+- kubernetes
+- machine-learning
+name: seldon-core-controller
```

Comment:
Change chart name "seldon-core-controller" to "seldon-core-operator". This will require all the related changes, eg this chart file's parent dir name itself

Reply:
Done